util/check_dif_statuses.py

```diff
@@ -76,24 +87,38 @@ def __init__(self, repo_top, difs_root_path, dif_name):
         self.dif_name = dif_name
         self.ip = self.dif_name[4:]
         self.dif_path = os.path.join(difs_root_path, dif_name)
-        self.hw_path = f"hw/ip/{self.ip}"
-        # Determine last date DIF and HW was updated.
-        self.dif_last_modified = self._get_last_commit_date(
-            repo_top, self.dif_path, [".h", ".c"])
+        # Check if header file exists - if not then its not even begun.
+        has_begun = os.path.isfile(self.dif_path + ".h")
+        self.hw_path = (f"hw/{top_level}/ip/{self.ip}"
+                        if self.ip in _TOP_LEVEL_IPS else f"hw/ip/{self.ip}")
+        # Indicates DIF API completeness.
+        self.num_functions_defined = -1
+        self.num_functions_implemented = -1
+        self.api_complete = False
+
+        # Determine last date HW was updated.
         self.hw_last_modified = self._get_last_commit_date(
             repo_top, os.path.join(self.hw_path, "rtl"), [""])
-        # Determine the main contributor of the DIF and HW.
-        self.dif_main_contributors = self._get_main_contributor_emails(
-            _OTComponent.DIF)
+        # Determine the main contributor of the HW.
         self.hw_main_contributors = self._get_main_contributor_emails(
             _OTComponent.HW)
-        # Determine lifecycle state
-        self.lifecycle_state = self._get_dif_lifecycle_state()
-        # Determine DIF API completeness.
-        self.num_functions_defined = -1
-        self.num_functions_implemented = -1
-        self.api_complete = False
-        self.funcs_unimplemented = self._get_funcs_unimplemented()
+        if has_begun:
+            # Determine last date DIF was updated.
+            self.dif_last_modified = self._get_last_commit_date(
+                repo_top, self.dif_path, [".h", ".c"])
+            # Determine the main contributor of the DIF.
+            self.dif_main_contributors = self._get_main_contributor_emails(
+                _OTComponent.DIF)
+            # Determine lifecycle state
+            self.lifecycle_state = self._get_dif_lifecycle_state()
+            # Determine DIF API completeness.
+            self.funcs_unimplemented = self._get_funcs_unimplemented()
+        else:
+            self.dif_last_modified = datetime.datetime.fromtimestamp(
+                datetime.MINYEAR, datetime.timezone.utc)
```

Comment:
This has a minor side effect - the last modified shows up as `1970-01-01`. The other alternative is to mark this as "N.A." or something, but the code in `print_status_table()` will need to be updated as well. Let me know what is preferred.

Reply:
I like the idea of marking it "N.A." or "-" to match the "contributors" and "lifecycle state" fields.

Otherwise, LTGM. Thanks for updating this!

Reply:
SG, updated. PTAL.